main.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime, timedelta
+
+import dash_bootstrap_components as dbc
+import pandas as pd
+import plotly.express as px
+from dash import Dash, Input, Output, State, dcc, html
+from dateutil.relativedelta import relativedelta
+from pandas.tseries.offsets import DateOffset
+
+from src import financial
+
+app = Dash(__name__, external_stylesheets=[dbc.themes.SANDSTONE])
+
+
+principal_input = html.Div(
+    [
+        dbc.Label("Montante inicial"),
+        dbc.Input(id="principal", value=1000, type="number"),
+    ],
+    className="mb-3",
+)
+
+rate_input = html.Div(
+    [
+        dbc.Label("Taxa anual (%)"),
+        dbc.Input(id="yearly_rate", value=10, type="number"),
+    ],
+    className="mb-3",
+)
+
+date_input = html.Div(
+    [
+        dbc.Label("Vencimento"),
+        html.Br(),
+        dcc.DatePickerSingle(
+            id="date", display_format="Y/MM/DD", date=datetime.now().date() + timedelta(days=365)
```

Comment:
```suggestion
            id="date", display_format="Y/MM/DD", date=datetime.now().date() + relativedelta(years=1, days=1)
```
Sugiro a troca, pois no formato anterior, o período inicial calculado sempre será menor que 12 meses (11 meses na func `get_month_difference`), ao acrescentar um dia no período garantimos que 12 meses passaram por completo.